app/routes/crate/version-dependencies.js

```diff
@@ -32,6 +34,17 @@ export default class VersionRoute extends Route {
     return version;
   }
 
+  async afterModel(_resolvedModel, transition) {
+    let crate = this.modelFor('crate');
+    // TODO: Resolved version without waiting for versions to be resolved
+    try {
+      await crate.loadVersionsTask.perform();
+    } catch (error) {
+      let title = `${crate.name}: Failed to load version data`;
+      return this.router.replaceWith('catch-all', { transition, error, title, tryAgain: true });
+    }
+  }
+
```

Comment:
I don't understand this part. what do we gain from moving this request to `afterModel()`? from my understanding `afterModel()` is also render-blocking? 🤔

Reply:
I can't quite remember the intention 😅  I think it might have been related to a commit I removed, to make the diffs between commits easier to read. I've reverted this part!